app/insomnia/api-specs.md

```diff
@@ -0,0 +1,88 @@
+---
+title: API specs in Insomnia
+
+content_type: concept
+layout: concept
+
+related_resources:
+  - text: Design APIs in Insomnia
+    url: /insomnia/design/
+  - text: Create a design document in Insomnia
+    url: /how-to/create-a-design-document/
+  - text: Generate a collection from a design document
+    url: /how-to/generate-a-collection-from-a-design-document/
+  - text: Lint API specs
+    url: /
+  - text: Automate linting
+    url: /
+  - text: Create a request
+    url: /how-to/create-a-request/
```

Comment:
@fabianrbz Oh, forgot to ask, I have this list of related resources here, but in the preview, the info box doesn't render with any information, including these. Any idea how I can fix this?

Reply:
@cloudjumpercat nice catch! I've pushed a fix to your branch.